stdlib/array.ml

```diff
@@ -439,10 +439,18 @@ let stable_sort cmp a =
 
 let fast_sort = stable_sort
 
+let shuffle_contract_violation i j =
+  let msg =
+    "Array.shuffle: rand(" ^ string_of_int (i + 1) ^ ")" ^
```

Comment:
That's not an idiomatic OCaml function call which is going to look weird when you get the error. I mean in `man` of my system `rand(3)` is actually a thing :-) If you feel parens are needed then I rather have `(rand %d)` and or use ocamldoc syntax `[rand %d]` but then we should establish a pattern here.

Reply:
Yet another way would be to use the compiler's convention for quoting code which I believe is straight quotes nowadays.